README.md

```diff
@@ -101,22 +108,253 @@ serial port name.
 
 ```js
 var ourBoard = new require('openbci-sdk').OpenBCIBoard();
-ourBoard.autoFindOpenBCIBoard(function(portName,ports) {
-    if(portName) {
+ourBoard.autoFindOpenBCIBoard().then((value) => {
+    if(Array.isArray(value)) {
```

Comment:
if upon not autofinding, it returns an empty array then this solution works just fine.  I was just trying to think about cases where autocomplete wouldn't work and how this statement would handle those cases. 

the solution i was proposing basically just says 'if we didn't find exactly 1 board, then do some other stuff.  i think yours should work fine though